utils/yoke/src/macro_impls.rs

```diff
@@ -57,6 +57,27 @@ impl_copy_type!(i128);
 impl_copy_type!(char);
 impl_copy_type!(bool);
 
+/// [`OwnedYokeable`] is a convenience type that allows one to wrap fully-owned types and produce
+/// [`Yokeable`]s from them. Its primary purpose is to be able to quickly satisfy the
+/// requirements on fields in the [`Yokeable`] and [`ZeroCopyFrom`] custom derive without
+/// needing to do a manual impl.
+///
+/// The [`ZeroCopyFrom`] impl will call `.clone()` since there is no borrowed version of this type. This
+/// will be cheap for `Copy` types.
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Ord, PartialOrd, Hash)]
```

Comment:
Suggestion: If you `#[derive(Copy)]` then I think you can call `.copy()` directly, and you can name the type `CopyYokeable`

Reply:
The derive is `if T: Copy`.

Reply:
(it's only `Copy if T: Copy`)